### PR TITLE
Added DAO Class Ticket implementation. The field of class Ticket were…

### PR DESCRIPTION
**Pull request by Zartemius**
… created and getters and setters for them.

### CODE REVIEW

File: src/main/java/entities/Ticket.java [hunk 1]
```diff
@@ -1,4 +1,62 @@
...
+
+    private int idTicket;
+    private int idFlight;
+    private int idCustomer;
+    private int Price;
+    private int LuggagePrice;
+    private boolean RegistrationPriority;
+
+    public int getIdTicket() {
+        return idTicket;
```
Comment (SMishAlex) [inline src/main/java/entities/Ticket.java]: I've already wrote that nameing like ticketId is better

File: src/main/java/entities/Ticket.java [hunk 2]
```diff
@@ -1,4 +1,62 @@
 package entities;
 
 public class Ticket {
+
+    private int idTicket;
+    private int idFlight;
+    private int idCustomer;
+    private int Price;
```
Comment (SMishAlex) [inline src/main/java/entities/Ticket.java]: it's better to use Long for price, we've discussed it on stand up

**Review by SMishAlex — COMMENTED** [review]
apply advice and make new commit here. dont merge it by yourself and dont create new brunch

**Comment by SMishAlex** [comment]
please use klysov's pull request after merge to add lombok, and chenge nameing like in all project now (without Impl in class' names)

**Review by bvvvd — COMMENTED** [review]
please use lombok to replace getters and setters by annotation

**Comment by Zartemius** [comment]
Kirill has already created a Ticket class that replaces mine. Close the pull request.

**Pull request closed without merging**